### DISCUSSION TITLE
unable to run makemigrations command

### DISCUSSION THREAD

**Discussion by satyam-pw** [netbox-community/netbox#16866] [Help Wanted!]
` root@netbox-testing-environment:/opt/netbox/netbox# python3 manage.py makemigrations`
CommandError: This command is available for development purposes only. It will
NOT resolve any issues with missing or unapplied migrations.

**Comment by candlerb** [comment]
What is the actual issue that you have, which makes you think that you need to run "makemigrations"?

As the message says, "makemigrations" is for people developing the Netbox code itself. If you are not a developer working on Netbox, you should not be using this command. If you *are* a Netbox developer, then you will know what this command does and how to bypass that restriction.
